src/crud.js

```diff
@@ -0,0 +1,11 @@
+import toDoTasks from './index.js'; // eslint-disable-line
+
+document.querySelector('.fa-angle-down').addEventListener('click', () => {
+  const todoData = document.querySelector('.newTodo').value;
+  if (todoData.trim() === '') {
+    return;
+  }
+  toDoTasks.push({ description: `${todoData}`, completed: false, index: toDoTasks.length });
+  localStorage.setItem('newTodo', JSON.stringify(toDoTasks));
+  window.location.reload();
```

Comment:
- [ ] The local storage is not saving the info, please 🙏🏽 fix it for a better user interface.

Reply:
@franclobo. thanks for your kind review. before submitting it was working well but what happened after that i am unable to understand. can you help me.